hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -4543,4 +4543,141 @@
       allowing for better identification and analysis of performance issues.
     </description>
   </property>
+
+  <property>
+    <name>hdds.datanode.dns.interface</name>
+    <value>default</value>
+    <tag>OZONE, DATANODE</tag>
+    <description>
+      The name of the Network Interface from which a data node should
```

Comment:
The descriptions here use various different spelling of Datanode. The new website only accepts "Datanode" so we should probably use that here as well.